apps/frontend/src/utils/helpers.ts

```diff
@@ -128,3 +128,9 @@ export const areAddressesEqual = (address1: string, address2: string) =>
 
 export const getChangelogUrl = (commit: string) =>
   `${GITHUB_LINKS.DAPP}/blob/${encodeURI(commit)}/apps/frontend/CHANGELOG.md`;
+
+export const calculateCollateralRatio = (
+  collateral: number,
+  debt: number,
+  price: number,
+) => ((collateral * price) / debt) * 100;
```

Comment:
```suggestion
export const calculateCollateralRatio = (
  collateral: Decimalish,
  debt: Decimalish,
  price: Decimalish,
) => decimalic(collateral).mul(price).div(debt).mul(100);
```

Use `Decimalish` from `@sovryn/utils` and `decimalic` helpers for values when doing calculations